index.html

```diff
@@ -0,0 +1,142 @@
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Microverse Trials</title>
+</head>
+<body>
+  <nav id="navbar">
+
+    <header>JS Documentation</header>
+    <ul>
+       <li><a class="nav-link" href="#Introduction">Introduction</a></li>
+       <li><a class="nav-link" href="#What_you_should_already_know">What you should already know</a></li>
+       <li><a class="nav-link" href="#JavaScript_and_Java">Javascript and Java</a></li>
+       <li><a class="nav-link" href="#Hello_world">Hello World</a></li>
+       <li><a class="nav-link" href="#Variables">Variables</a></li>
+    </ul>
+    
+ </nav>
+
+
+ <main id="main-doc">
+
+   <section class="main-section" id="Introduction">
+     <h1>Introduction</h1>
+     <p>
+       JavaScript is a cross-platform, object-oriented scripting
+       language. It is a small and lightweight language.
+       Inside a host environment (for example, a web browser),
+        JavaScript can be connected to the objects of its environment to provide
+         programmatic control over them.
+     </p>
+     <p>
+       JavaScript contains a standard library of objects,
+        such as Array, Data and Math, and a core set of language elements such as
+      operators, control structures, and statements.
+      Core JavaScript can be extended for a variety of purposes by
+      supplementing it with additional objects; for example:
+     </p>
+     <ul>
+       <li>
+         Client-side JavaScript extends the core language by supplying objects
+         to control a browser and its Document Object Model (DOM). For example, client-
+         side extensions allow an application to place elements on an HTML form and
+         respond to user events such as mouse clicks, form input, and page navigation.
+       </li>
+       <li>
+         Server-side JavaScript extends the core language by supplying objects relevant
+         to running JavaScript on a server. For example, server-side extensions allow an
+          application to communicate with a database, provide continuity of information
+          from one invocation to another of the application, or perform file manipulations
+           on a server.
+       </li>
+     </ul>
+   </section>
+
+   <section class="main-section" id="What_you_should_already_know">
+     <h1>What you should already know</h1>
+     <p>This guide assumes you have the following basic background:</p>
+     <ul>
+       <li>
+         A general understanding of the Internet and the World Wide Web (WWW).
+       </li>
+       <li>
+         Good working knowledge of HyperText Markup Language (HTML).
+       </li>
+       <li>
+         Some programming experience. If you are new to programming,
+          try one of the tutorials linked on the main page about JavaScript.
+       </li>
+     </ul>
```

Comment:
it would be a good idea to style this list to improve user experience. You can even add icons inline with the text if you like

Reply:
* Good job stating why the requested change is important 👍